.circleci/config.yml

```diff
@@ -10,70 +11,73 @@ jobs:
       tag: "17.0"
     steps:
       - checkout
-      - run: mkdir -p ~/build-artifacts
+      - run: mkdir -p ./build-artifacts
       - gradle/with_cache:
           steps:
             - run: ./gradlew build -x test
       - run:
-          name: Copy JARs to build-artifacts
-          command: |
-            shopt -s globstar
-            cp **/build/libs/*.jar ~/build-artifacts
+          name: Show the JARs
+          command: ls -l build-artifacts
```

Comment:
Gradle has been configured to send JARs to `./build-artifacts` when built, so CI does not need to copy them anymore. Show them instead!